integration/update_field_compat_test.go

```diff
@@ -793,3 +794,121 @@ func TestUpdateFieldCompatMixed(t *testing.T) {
 
 	testUpdateCompat(t, testCases)
 }
+
+func TestUpdateFieldCompatMul(t *testing.T) {
+	t.Parallel()
+
+	providers := shareddata.AllProviders().
+		Remove("BigDoubles").
```

Comment:
For each call of `Remove` let's add a comment with a reason why we remove it.

Reply:
Applied :+1: